force-app/main/default/classes/ApplicationRegistration.cls

```diff
@@ -34,20 +34,24 @@ public class ApplicationRegistration {
             }
 
             // Create questions for first name, middle name, last name, email, password, and password confirmation
+            // Required__c is usually tied to Required__c = appCreate.app.Application_Status__c, but that won't work here, since the app isn't created properly until appCreate.applicationCreateSetup(). -JJH
             ApplicationQuestion firstName = new ApplicationQuestion(appCreate.con, appCreate.app, '', interaction, 'First_Name__c');
-            firstName.addQuestionObject(new Question__c(Label__c = 'First Name', Required__c = appCreate.app.Application_Status__c, Active__c = true));
+            firstName.addQuestionObject(new Question__c(Label__c = 'First Name', Required__c = 'yep', Active__c = true));
```

Comment:
@jjheaney, Where is 'yep' used and what was the purpose of changing this?

Reply:
This is explained in a the comment I added in the code a few lines above:

"Required__c is usually tied to Required__c = appCreate.app.Application_Status__c, but that won't work here, since the app isn't created properly until appCreate.applicationCreateSetup() is called. -JJH"

All the item requires to become required is _any_ value in this Required__c field. I considered using "yes," but I thought that would imply that changing "yes" to "no" would change the item from Required to Optional. That's not the case; "no" would still be a value in Required__c and so the item would remain required. I opted for "yep" because it both correctly communicates that the field is required and is bizarrely informal enough that any coder reading it will go "wait, what?" and investigate the rules for the field before making any changes.

Looks like a dumb mistake I made in 30 seconds -- so fair enough asking about it -- but actually a really calculated change I planned for hours to make as clear and riskless as possible!